src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+set(LIBSLEEF_SRCS sleefdp.c sleefsp.c)
+
+add_library(sleef SHARED ${LIBSLEEF_SRCS})
```

Comment:
Please define a variable that carries the name of the target. 

If possible, I'd rather have all the variables that carry target (libraries and executables) in the root folder cmake script, each of them with a description of what the target is and where the add_[executable|library] command is located in the project. I think this is better than having targets spread around in the project tree tree.